content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1790,6 +1790,27 @@ uses this annotation.
 See [annotations](https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/guide/service/annotations/)
 in the AWS load balancer controller documentation.
 
+### service.beta.kubernetes.io/aws-load-balancer-security-groups (deprecated) {#service-beta-kubernetes-io-aws-load-balancer-security-groups}
+
+Example: `service.beta.kubernetes.io/aws-load-balancer-security-groups: "sg-53fae93f,sg-8725gr62r"`
+
+Used on: Service
+
+The AWS load balancer controller uses this annotation to specify a comma seperated list
+of frontend securityGroups you want to attach to an NLB. Both name and ID of securityGroups
```

Comment:
```suggestion
of  security groups you want to attach to an AWS load balancer. Both name and ID of security
```

Reply:
I wouldn't specify in _Kubernetes_ docs what kind of load balancer you get. That detail may change through work outside of Kubernetes.